src/AI/agents/agents/services/events/jobs.py

```
@@ -0,0 +1,74 @@
+                "last_message": existing.get("last_message"),  # Preserve the assistant message
+            }
+        
+        for cb in self._subs.get(event.session_id, []):
+            try:
+                result = cb(event)
+                # If callback is async, schedule it properly
+                if asyncio.iscoroutine(result):
+                    try:
+                        # Try to get the running loop
+                        loop = asyncio.get_running_loop()
+                        loop.create_task(result)
```

Ruff (0.14.8)

Store a reference to the return value of `loop.create_task`

(RUF006)